services/provider/server/server.go

```diff
@@ -418,6 +426,21 @@ func (s *OCSProviderServer) getExternalResources(ctx context.Context, consumerRe
 
 	}
 
+	//Enable CSI-OMAP is storageClusterPeer is present
+	clientMappingConfig, err := s.getClientMapping(ctx)
+	if err != nil {
+		return nil, err
+	}
+	_, hasClientID := clientMappingConfig.Data[string(consumerResource.UID)]
+
+	extR = append(extR, &pb.ExternalResource{
+		Name: "ocs-ceph-csi-config",
+		Kind: "ConfigMap",
+		Data: mustMarshal(map[string]string{
+			"CSI_ENABLE_OMAP_GENERATOR": strconv.FormatBool(hasClientID),
+		}),
+	})
```

Comment:
What is this ocs-ceph-csi-config? I believe that ceph csi operator is using a CR for that